opendcs-rest-api-jetty/build.gradle

```diff
@@ -46,6 +46,21 @@ task startJetty(type: JavaExec) { ->
             "-s")
 }
 
+task copySwagger(type: Copy) {
```

Comment:
I'm not sure what this task is doing, given the war/jar for the opendcs-rest-api already include these resources. Also this looks to be copying into a tmp directory, but I don't see where that tmp directory is used.